### PR TITLE
[device_info_plus] Add TizenDeviceInfo.data property

### PR DESCRIPTION
**Pull request by swift-kim**
- Add `TizenDeviceInfo.data` which represents the device info as a map. ([source](https://github.com/fluttercommunity/plus_plugins/blob/a246de0323c454b0a89dfa0fc1ba6e7aa58ef477/packages/device_info_plus/device_info_plus_platform_interface/lib/model/base_device_info.dart#L7))
- Disambiguate the method channel name to avoid a possible conflict with the original device_info_plus.

### CODE REVIEW

File: packages/device_info_plus/lib/device_info_plus_tizen.dart [hunk 1]
```diff
@@ -109,12 +109,33 @@ class TizenDeviceInfo {
...
+        'buildTime': buildTime,
+        'buildType': buildType,
+        'buildVariant': buildVariant,
+        'buildRelease': buildRelease,
+        'deviceType': deviceType,
+        'manufacturer': manufacturer,
+        'platformName': platformName,
+        'platformProcessor': platformProcessor,
+        'tizenId': tizenId,
+      };
```
Comment (JSUYA) [inline packages/device_info_plus/lib/device_info_plus_tizen.dart:133]: Can you explain where this data getter is used? I couldn't find out why this is needed.

Comment (JSUYA) [reply packages/device_info_plus/lib/device_info_plus_tizen.dart:133]: Thank you for explain. I missed PR comment.
I think we need to add the toString() API too.

https://github.com/fluttercommunity/plus_plugins/blob/a246de0323c454b0a89dfa0fc1ba6e7aa58ef477/packages/device_info_plus/device_info_plus_platform_interface/lib/model/base_device_info.dart#L13-L16

Comment (swift-kim) [reply packages/device_info_plus/lib/device_info_plus_tizen.dart:133]: Okay, let me clean up a bit.

File: packages/device_info_plus/lib/device_info_plus_tizen.dart [hunk 2]
```diff
@@ -10,8 +10,8 @@ import 'package:flutter/services.dart';
 ///
 /// See: https://docs.tizen.org/application/native/guides/device/system
 class TizenDeviceInfo {
-  /// Tizen device info class.
-  TizenDeviceInfo({
+  TizenDeviceInfo._({
```
Comment (swift-kim) [inline packages/device_info_plus/lib/device_info_plus_tizen.dart:13]: Technically this is a breaking change but I don't think there's any use case.

File: packages/device_info_plus/lib/device_info_plus_tizen.dart (hunk 1 above)
Comment (swift-kim) [reply packages/device_info_plus/lib/device_info_plus_tizen.dart]: Done. Thanks!

File: packages/device_info_plus/lib/device_info_plus_tizen.dart (hunk 2 above)
Comment (JSUYA) [reply packages/device_info_plus/lib/device_info_plus_tizen.dart:13]: Just curious! is there a reason why a named constructor is necessary here technically? kind of pattern?

**Review by JSUYA — APPROVED** [review]

File: packages/device_info_plus/lib/device_info_plus_tizen.dart (hunk 2 above)
Comment (swift-kim) [reply packages/device_info_plus/lib/device_info_plus_tizen.dart:13]: This just changes the visibility of the constructor. Any member that starts with `_` is considered private in Dart.

**Pull request merged**